.github/workflows/check-service-labels.yml

```diff
@@ -13,8 +13,10 @@ jobs:
   check-labels:
     runs-on: ubuntu-latest
     steps:
-    - name: Check servicing labels
+    - name: Check `Servicing-approved` label
       run: |
+        echo "Merging permission is enabled for servicing PRs when the `Servicing-approved` label is applied."
+        echo "Rules for applying the label can be found [here](~/docs/project/library-servicing.md)."
```

Comment:
- Markdown notation isn't particularly useful in this context. 
- That MD doesn't mention this check, nor have anything in the context of approved, nor is it library specific. Is that the best place to have it in?

Reply:
I can use the full link to a specific version of that file.

I added those new notes in this PR: https://github.com/dotnet/runtime/pull/85731/files#diff-a15dd9f7cefdcca5f2996aff0f33d60ccb4189dcd72c42edd9f9014a234f82c7

Reply:
And I see what you mean about using markdown. Bummer.

![image](https://user-images.githubusercontent.com/1175054/236069436-618b7c01-cc06-48d7-b53a-210231b2d859.png)

Reply:
Replacing with a full github path pointing to main:

```suggestion
        echo "Rules for applying the label can be found in https://github.com/dotnet/runtime/blob/main/docs/project/library-servicing.md"
```